test/Libraries/DynamoMSOfficeTests/Properties/AssemblyInfo.cs

```diff
@@ -1,10 +1,18 @@
-﻿using System.Reflection;
+using NUnit.Framework;
+using System.Reflection;
 using System.Runtime.InteropServices;
+using System.Threading;
 
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("DynamoMSOfficeTests")]
 [assembly: AssemblyCulture("")]
+
+#if NET6_0_OR_GREATER
```

Comment:
what is this about ?

Reply:
It is a condition needed for build test in net60 and net48.  Some NUnit tests need to run in STA threads.

Reply:
But why is it in AssemblyInfo.cs ?
Is this a good place to put this code ?
Should we set the ApartmentState in the `OneTimeSetUp` code of the tests ?
Also how come `[assembly: RequiresSTA]` was not there before on the net48 target ? Why do we need it now. ?

Reply:
You're right, I removed.